charts/jaeger/templates/query-ing.yaml

```diff
@@ -1,4 +1,6 @@
 {{- if .Values.query.ingress.enabled -}}
+{{- $ingressApiIsStable := eq (include "jaeger.ingress.isStable" .) "true" }}
```

Comment:
it should be removed as this is not used in this file

Reply:
done